pkg/api/org_users.go

```diff
@@ -190,64 +190,69 @@ func (hs *HTTPServer) GetOrgUsers(c *models.ReqContext) response.Response {
 		return response.Error(http.StatusBadRequest, "orgId is invalid", err)
 	}
 
-	result, err := hs.getOrgUsersHelper(c, &org.GetOrgUsersQuery{
+	result, err := hs.searchOrgUsersHelper(c, &org.SearchOrgUsersQuery{
 		OrgID: orgId,
 		Query: "",
 		Limit: 0,
 		User:  c.SignedInUser,
-	}, c.SignedInUser)
+	})
 
 	if err != nil {
 		return response.Error(500, "Failed to get users for organization", err)
 	}
 
-	return response.JSON(http.StatusOK, result)
+	return response.JSON(http.StatusOK, result.OrgUsers)
 }
 
-func (hs *HTTPServer) getOrgUsersHelper(c *models.ReqContext, query *org.GetOrgUsersQuery, signedInUser *user.SignedInUser) ([]*org.OrgUserDTO, error) {
-	result, err := hs.orgService.GetOrgUsers(c.Req.Context(), query)
+// swagger:route GET /orgs/{org_id}/users/search orgs searchOrgUsers
+//
+// Search Users in Organization.
+//
+// If you are running Grafana Enterprise and have Fine-grained access control enabled
+// you need to have a permission with action: `org.users:read` with scope `users:*`.
+//
+// Security:
+// - basic:
+//
+// Responses:
+// 200: searchOrgUsersResponse
+// 401: unauthorisedError
+// 403: forbiddenError
+// 500: internalServerError
+func (hs *HTTPServer) SearchOrgUsers(c *models.ReqContext) response.Response {
+	orgId, err := strconv.ParseInt(web.Params(c.Req)[":orgId"], 10, 64)
```

Comment:
nit: `orgID` instead of orgId

Reply:
It's how it's specified in org route, so if we want to update it, it should be updated everywhere, otherwise API will be broken:
https://github.com/grafana/grafana/blob/bd9cce2866ff507e95b62dd32b73c0a25394f342/pkg/api/api.go#L355

Reply:
I just meant the variable name

Reply:
Ah, that's ok